### PR TITLE
Highlight search string matches in bold

### LINKED ISSUE #149: Highlight search string matches in bold
Once #147 is merged. Like we do in Applications Menu.

### PR DESCRIPTION
**Pull request by alainm23**
Fixed #149

image

### CODE REVIEW

File: src/SearchView.vala [hunk 1]
```diff
@@ -179,5 +198,54 @@ public class Switchboard.SearchView : Gtk.Box {
...
+            if (pattern == "") {
+                return MARKUP.printf (Markup.escape_text (text));
+            }
+
+            // if no text found, use pattern
+            if (text == "") {
+                return MARKUP.printf (Markup.escape_text (pattern));
+            }
+
+            var matchers = Synapse.Query.get_matchers_for_query (
```
Comment (danirabbit) [inline src/SearchView.vala:214]: Seems a bit overkill maybe :sweat_smile:

Comment (alainm23) [reply src/SearchView.vala:214]: I copied the code from the application menu 😅, but I will look for a way to optimize the code.

**Comment by alainm23** [comment]
I further optimized the code, it works in all the poor scenarios.

**Review by danirabbit — APPROVED** [review]
Nice, way more concise haha. Great job!

**Pull request merged**